.expeditor/publish-release-notes.sh

```diff
@@ -24,19 +24,35 @@ pushd ./automate.wiki
 ## Improvements
 -
 
+## Compliance Profile Updates
+-
+
 ## Bug Fixes
 -
 
+## Maintenance
+-
+
 ## Backward Incompatibilities
 -
 
 ## Chef Product Versions
 
 This release uses:
-- Chef Habitat version
-- Chef Habitat Builder version
-- Chef Infra Server version
-- Chef InSpec version
+- Chef Habitat version:
+- Chef Habitat Builder version:
+- Chef Infra Server version:
+- Chef InSpec version:
+
+## Service Versions
+
+This release uses:
+- Postgres:
+- ElasticSearch:
+- Nginx:
+- Haproxy:
```

Comment:
Code base search would suggest this capitalization:
```suggestion
- HAProxy:
```